lib/angular/portal.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = function(mainModule, userConfig) {
+  mainModule.config(function($stateProvider) {
+    if (userConfig && userConfig.mobile) {
+      return;
+    }
+
+    $stateProvider
+      .state('app.map', {
+        url: '/map',
+        resolve: {
+          workorders: function(workorderManager) {
+            return workorderManager.list();
+          }
+        },
+        data: {
+          columns: 2
+        },
+        views: {
+          content: {
+            template: '<workorder-map workorders="ctrl.workorders" center="ctrl.center" container-selector="#content"></workorder-map>',
```

Comment:
The template for mobile and portal are almost identical. I don't think we need both.